src/modules/home/home.route.tsx

```
@@ -13,4 +16,14 @@ export const homeRoutes = [
     path={homePaths.home}
     element={<UserHomeView />}
   />,
+  <Route
+    id={homePaths.details}
+    path={homePaths.details}
+    element={<ProductDetails />}
+  />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)